test/appRelationsDiscovery_test.js

```diff
@@ -0,0 +1,16 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+const expect = chai.expect;
+chai.use(chaiAsPromised);
+
+const {getRelations} = require('../lib/appRelationsDiscovery');
+const {getAsyncApiExamples} = require('./testsUtil'); 
+
+describe('appRelationDiscovery', function() {
+  it('should return a map of relations', async function() {
```

Comment:
you are not really checking if it returns a map of relations but just if returns object.
you should have a real map here in the test and check the output against this "manually" prepared map

Reply:
Yeah, there was an error coming on running the tests. So this is just an initial setup.